### PR TITLE
Update test for https://github.com/apple/swift/pull/74201

### PR DESCRIPTION
**Pull request by slavapestov**

### CODE REVIEW

**Comment by slavapestov** [comment]
https://github.com/apple/swift/pull/74299
@swift-ci Please test

**Review by adrian-prantl — APPROVED** [review]

File: lldb/test/API/lang/swift/macro/TestSwiftMacro.py [hunk 1]
```diff
@@ -42,8 +42,8 @@ def testDebugging(self):
         thread.StepInto()
         # This is the expanded macro source, we should be able to step into it.
         self.expect('reg read pc', substrs=[
-            '[inlined] freestanding macro expansion #1 of stringify',
-            '13testStringify'
+            '[inlined] freestanding macro expansion #1 of stringify in module a file main.swift line 5 column 11',
+            '__swiftmacro_1a0015mainswift_tzEGbfMX4_10_9stringifyfMf_'
```
Comment (adrian-prantl) [inline lldb/test/API/lang/swift/macro/TestSwiftMacro.py]:
```suggestion
            'stringify'
```
This should suffice for what we're interested in here, and should be more future proof.

**Comment by slavapestov** [comment]
https://github.com/apple/swift/pull/74299
@swift-ci Please test

**Review by bnbarham — APPROVED** [review]

File: lldb/test/API/lang/swift/macro/TestSwiftMacro.py (hunk 1 above)
Comment (bnbarham) [reply lldb/test/API/lang/swift/macro/TestSwiftMacro.py]: @adrian-prantl did you also mean removing the `in module a file...`?

Comment (adrian-prantl) [reply lldb/test/API/lang/swift/macro/TestSwiftMacro.py]: no, just the details of the mangled name that are more likely to change

**Pull request merged**